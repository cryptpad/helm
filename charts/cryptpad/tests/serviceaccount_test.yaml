templates:
  - serviceaccount.yaml
tests:
  - it: Assert the service match the snapshot
    asserts:
      - matchSnapshot:
          path: metadata