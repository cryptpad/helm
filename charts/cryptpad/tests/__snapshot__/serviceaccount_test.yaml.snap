Assert the service match the snapshot:
  1: |
    labels:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/managed-by: Helm
      app.kubernetes.io/name: cryptpad
      helm.sh/chart: cryptpad-0.0.2
    name: RELEASE-NAME-cryptpad
